owners.html

```diff
@@ -1,161 +1,151 @@
 <!DOCTYPE html>
 <html lang="ru">
-  <head>
```

Comment:
Currently we don't use any lintner in our project nevetheless before the moment we'll start I'm not able to apply so many changes which related to modifying and refactoring of code regarding to htmllint rules.

Reply:
If I understand you correctly, we're not allowed to refactor structure of the code at the moment. So I'll revert affected lines to the default structure. @MaxFlower